docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3.8'
 services:
   # Dependencies
   postgres:
-    image: postgres:15
+    image: postgres:16
```

Comment:
🤷‍♂️

Reply:
I figured tracking the latest stable will allow us to get ahead of any incompatibilities quickly. WDYT @AndrewDryga? Better to track the version on GCP?

https://www.postgresql.org/about/news/postgresql-16-released-2715/